kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -24,10 +24,17 @@ const publicMethods = [
  * is configured in webpack.config.js
  * @classdesc
  */
-function Lib() {
+function Lib(kolibriGlobal) {
   // libraries
   this.loglevel = require('loglevel');
   this.vue = require('vue');
+  this.vue.mixin({
+    methods: {
+      url: (url_name, ...args) => {
+        return kolibriGlobal.urls[url_name](...args);
+      }
```

Comment:
This seems okay to me. Though since it's a global mixin on all Vue instances, then you should try to both minimize the surprise to future devs and "publicize" this feature.

Maybe an alternative to a global mixin is to make Vue instances use `kolibriGlobal.urls`? Should be fine as long as the module is loaded *after* the core js app which defines `kolibriGlobal`.